raster/r.out.vtk/r.out.vtk.html

```diff
@@ -97,11 +97,11 @@ <h3>Spearfish example with RGB data</h3>
 g.region n=4926990 s=4914840 w=591570 e=607800 res=30 -p
 
 # using r.in.wms to create RGB data to get a satellite coverage
-r.in.wms layers=global_mosaic mapserver=http://wms.jpl.nasa.gov/wms.cgi \
-         output=wms_global_mosaic
+r.in.wms layers=BlueMarbleNG mapserver=https://gibs.earthdata.nasa.gov/wms/epsg4326/best/wms.cgi?	
```

Comment:
```suggestion
r.in.wms layers=BlueMarbleNG url=https://gibs.earthdata.nasa.gov/wms/epsg4326/best/wms.cgi? \
```

Reply:
Reasoning behind:

1) Missing `\` at the end of the line, making `sh` understand it as two one-line commands
2) Changing `mapserver` to `url` in `r.in.wms`. The change happened somewhere between GRASS 6 and GRASS 7 and it is a prehistoric remnant that should not be there for a long time, already.

Reply:
Fixed in 27412da89c99c712f71660d3c1fa653465844348